hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/counter/zookeeper/ZookeeperCounterReporter.java

```diff
@@ -93,7 +93,7 @@ private static TopicName escapedTopicName(String qualifiedTopicName) {
         TopicName topicName = fromQualifiedName(qualifiedTopicName);
         return new TopicName(
                 escapeMetricsReplacementChar(topicName.getGroupName()),
-                escapeMetricsReplacementChar(topicName.getName())
+                topicName.getName()
```

Comment:
okay, one more: why do we fix topic name and not group name? what if group name has an underscore?